Fix npm run build error

**Actual Behaviour**

Error on running command: `npm run build`

**Expected Behaviour**

Should be eliminated

**Steps to reproduce it**

npm run build on any latest chai susi

**LogCat for the issue**

> ➜  chat.susi.ai git:(master) npm run build

> chat.susi.ai@0.1.0 build /Users/shubhamgupta/Desktop/chat/chat.susi.ai
> react-scripts build

> Creating an optimized production build...
> Failed to compile.
> 
> static/js/main.618d1eb1.js from UglifyJs
> SyntaxError: Unexpected token: operator (>) [./~/query-string/index.js:8,0]
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! chat.susi.ai@0.1.0 build: `react-scripts build`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the chat.susi.ai@0.1.0 build script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/shubhamgupta/.npm/_logs/2018-12-01T23_39_07_197Z-debug.log

**Screenshots of the issue**

<img width="1088" alt="screenshot 2018-12-02 at 5 09 29 am" src="https://user-images.githubusercontent.com/18073126/49334028-c9b77600-f5f0-11e8-98f2-12f0a92afeee.png">

**Would you like to work on the issue?**

Yes

--- a/package.json
+++ b/package.json
@@ -34,7 +34,6 @@
     "moment": "2.22.2",
     "npm": "6.3.0",
     "prop-types": "15.6.2",
-    "query-string": "6.2.0",
     "react": "15.6.2",
     "react-bootstrap": "0.31.5",
     "react-color": "2.14.1",

--- a/src/helpers/ajax.js
+++ b/src/helpers/ajax.js
@@ -1,6 +1,5 @@
 import axios from 'axios';
 import _ from 'lodash';
-import queryString from 'query-string';
 import { camelizeKeys, toFormData } from '../utils';
 import Cookies from 'universal-cookie';
 
@@ -43,9 +42,9 @@ const obj = {};
         };
       } else if (method === 'get') {
         if (payload) {
-          url += `?${queryString.stringify(payload, {
-            encode: false,
-          })}`;
+          url += `?${Object.keys(payload)
+            .map(key => key + '=' + payload[key])
+            .join('&')}`;
         }
       }
 

